### PR TITLE
Fixes https://github.com/tpbowden/swarm-ingress-router/issues/33

### PR DESCRIPTION
**Pull request by ernest-okot**
:sweat_smile: Never written in Golang before, but here we go

### CODE REVIEW

**Review by tpbowden — APPROVED** [review]

**Comment by tpbowden** [comment]
Is this still WIP? Looks fine to me.

**Comment by ernest-okot** [comment]
Was waiting for your approval, please merge whenever you can

**Pull request merged**
